docs/UserGuide.md

```diff
@@ -490,18 +509,23 @@ Otherwise, it is only removed from the specified contact/task label list.
 Multiple labels can be deleted in the same command. 
 However, only a maximum of one contact and one task can be edited within the same command.
 
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
+deleteL is case-sensitive! If you try to delete `cs2103t` but type `CS2103T` in your command, 
+you will encounter an error as YellowBook will not find a match.
+</div>
+
 The following scenarios should not happen for your command to run successfully: 
 - Contact/task does not exist
 - Label does not exist on specified contact/task
 - No contact/task is provided
 - No label is provided
 - More than one contact or more than one task is specified
 
-Format: `deleteL t/INDEX l/label_NAME`
+Format: `deleteL c/INDEX t/INDEX l/label_NAME`
 
 Example:
 
-* `deleteL t/14 l/CS2101` will remove the label "CS2101" from the 14th task on the task list.
+* `deleteL c/12 t/14 l/CS2101` will remove the label "CS2101" from the 14th task on the task list.
```

Comment:
Shouldn't `deleteL` also remove the label from the 12th task if not there will be an error as the label does not exist on the task?